tool/gravity/cli/teleport.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/gravitational/gravity/lib/loc"
+	"github.com/gravitational/gravity/lib/localenv"
+	"github.com/gravitational/gravity/lib/pack"
+
+	"github.com/gravitational/teleport/lib/config"
+	"github.com/gravitational/teleport/lib/defaults"
+
+	"github.com/gravitational/trace"
+	"gopkg.in/yaml.v2"
+)
+
+func showTeleportConfig(env *localenv.LocalEnvironment, packageName string) error {
+	locators, err := getTeleportLocators(env, packageName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	config, err := readTeleportFileConfig(locators.configReader)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	configBytes, err := yaml.Marshal(config)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	fmt.Println(string(configBytes))
+	return nil
+}
+
+func updateTeleportMasterTokens(env *localenv.LocalEnvironment, packageName string, tokens []string) error {
+	locators, err := getTeleportLocators(env, packageName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	fileConfig, err := readTeleportFileConfig(locators.configReader)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	for _, token := range tokens {
+		fileConfig.Auth.StaticTokens = append(fileConfig.Auth.StaticTokens, config.StaticToken(
+			fmt.Sprintf("node:%v", token)))
+	}
+	err = saveTeleportFileConfig(env.Packages, fileConfig, locators.teleportLocator, locators.configLocator, locators.configEnvelope.RuntimeLabels)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	fmt.Println("Teleport master auth token updated. Please restart gravity-site using 'kubectl -nkube-system delete pods -lapp=gravity-site'.")
```

Comment:
https://github.com/gravitational/gravity/blob/version/5.5.x/tool/gravity/cli/rotate.go#L90

Reply:
I'd rather keep this atomic, so these methods are only for updating config. Users can restart teleport/gravity-site themselves. I was planning on writing a KB article on how to use these if needed.